core/src/test/scala/com/carto/analyticstoolbox/HiveTestEnvironment.scala

```diff
@@ -70,6 +70,7 @@ trait HiveTestEnvironment extends TestEnvironment { self: Suite with BeforeAndAf
       .set("spark.hadoopRDD.ignoreEmptySplits", "false")
       .set("spark.sql.warehouse.dir", warehouseDir)
       .set("javax.jdo.option.ConnectionURL", derbyConnectionURL)
+      .set("spark.sql.extensions", "com.carto.analyticstoolbox.spark.SpatialFilterPushdownOptimizations")
```

Comment:
`.set("spark.sql.extensions", classOf[SpatialFilterPushdownOptimizations].getName)` in case we'll move packages around, it will be more convenient to maintain.

Reply:
However, whith this change, the double registration of this optimization happens; how about commenting out the body of the [registerOptimizations function](https://github.com/CartoDB/analytics-toolbox-databricks/blob/dbe303e60b85f61597820c36e90f6160d47b8935/core/src/test/scala/com/carto/analyticstoolbox/HiveTestEnvironment.scala#L45-L46):

```scala
// before
def registerOptimizations(sqlContext: SQLContext): Unit =
    SpatialFilterPushdownRules.registerOptimizations(sqlContext)
    
// after 
def registerOptimizations(sqlContext: SQLContext): Unit = {}
```

Reply:
Why even keep that function around?

Reply:
@moradology my idea was that we don't need these optimizations for some of the tests (thats a good test by itself btw). While I'm writing it, I'm realizing that we probably need to remove `spark.sql.extensions` completely / add it in a very specific test that has a changed Spark configuration to check that it operates as expected.